src/plcc/load_spec/validate_spec/validate_syntactic_spec/resolve_repeating_rules.py

```diff
@@ -0,0 +1,103 @@
+from typing import List
+from plcc.load_spec.load_rough_spec.parse_lines import Line
+from plcc.load_spec.parse_spec.parse_syntactic_spec.structs import (
+    LhsNonTerminal,
+    RhsNonTerminal,
+    StandardSyntacticRule,
+    Symbol,
+    SyntacticSpec,
+    RepeatingSyntacticRule,
+)
+
+
+def resolve_repeating_rules(spec: SyntacticSpec):
```

Comment:
I like your "commit" message on the PR, and it makes me think we should rename this module. `resolve` is a bit vague. It doesn't tell us what's happening. How about `transform`, `convert`, or `replace`? And then we probably need to say what it's being transformed/converted/replaced with. So maybe...

* `replace_repeating_rules_with_standard_rules`

We could make it a little shorter by sharing `rules` ...

* `replace_repeating_with_standard_rules`

I don't think we can drop `rules` without loosing too much context.

OK, I'm convinced. Please rename to `replace_repeating_with_standard_rules`. This includes the file name. Sorry.

Please rename anything else in here to make things more consistent with the new name. Again. Sorry.